docs/blog/microbit/2020-release.md

```diff
@@ -0,0 +1,81 @@
+
+https://youtu.be/G3bW_AMyDHA
+
+### Debugger
+
+Many Computing Education Researchers define knowledge of a programming language to be the mapping between syntax, semantics, and state. I think all too often we focus on syntax and semantics in our educational tools. No more! Debuggers are critical tools to help students understand the state of their program execution flow. We’ve worked hard to add debug capabilities to both Block and Text editors. 
+
+![Debugger](/static/blog/microbit/2020-release/debugger.gif)
+
+### GitHub Projects
+
+We’ve been working with our friends at GitHub over the past several months to provide a way to store MakeCode projects in a GitHub repository. This is great for group projects, version control, and teaching students about open source software development. Learn more about this feature here https://makecode.com/blog/makecode-with-github. 
+
+![GitHub](/static/blog/microbit/2020-release/github.jpg)
+
+### Web USB
+
+Could it be? A bird, a plane…? Better than superman, it’s Web USB to the rescue! No more dragging-and-dropping program files!! Web USB technology, allows you to ‘pair’ USB devices to your web browser, so when you click that Download button, it really does download directly to the micro:bit! You will need the latest firmware on your micro:bit device, and you’ll need to be using either new Edge or Chrome browsers. 
+
+![WebUSB](/static/blog/microbit/2020-release/webusb.gif)
+
+### More Music!
+
+We’ve added a couple new blocks to the Music category. Set Volume was very frequently requested by Teachers in classrooms full of micro:bits and speakers! And the Play Melody block was created by our amazing intern Kim (https://www.linkedin.com/in/kimberlymying/), and we’re excited to bring it to the micro:bit and see what lovely melodies you all create with it! 
+
+![Music](/static/blog/microbit/2020-release/music.gif)
+
+### Break & Continue
+
+These blocks can be found in the Loops category and allow you to exit out of long-running loops, or continue processing after a check or pause. 
+
+![Break and continue blocks](/static/blog/microbit/2020-release/break-continue.jpg)
+
+### Rudolph connectors
+
+That’s what we call these indicators we’ve added to help dragging and dropping embedded blocks to the right place. 
+
+![Rudolph connectors on blocks](/static/blog/microbit/2020-release/rudolph.gif)
+
+### Prompt for Project Name
+
+A last little feature we’ve added to help alleviate the hundreds of “Untitled” projects floating around, is a prompt to name projects as soon as a student clicks the New Project button. 
+
+![](/static/blog/microbit/2020-release/create-project.jpg)
+
+We hope these new features and improvements are helpful. Please let us know what you think. 
+
+Happy Making and Coding!
```

Comment:
Dual editor support

Using https://MakeCode.microbit.org/multi you can author 2 different radio Microbit programs at the same time.

Reply:
I couldn't demo this since we haven't bumped the cloud yet.  Feel free to add this feature, and create a gif...